karaf/templates/Dockerfile

```diff
@@ -32,18 +31,14 @@ RUN echo securerandom.source=file:/dev/urandom >> /usr/lib/jvm/java/jre/lib/secu
 # Add jboss user to the root group
 RUN usermod -g root -G jboss jboss
 
-{{? fp.param.base === "rhel" }}
-# Install Maven via RPM
-ADD "http://download.eng.bos.redhat.com/brewroot/packages/maven3/{{= fp.config.base.lib.version.maven.split("-")[0] }}/1.el7/noarch/maven3-${MAVEN_VERSION}.noarch.rpm" /tmp/
-RUN rpm -i "/tmp/maven3-${MAVEN_VERSION}.noarch.rpm" \
- && rm "/tmp/maven3-${MAVEN_VERSION}.noarch.rpm"
+
+# Install Maven via SCL
+{{? fp.param.base === "rhel"}}
+RUN yum-config-manager --enable rhel-server-rhscl-7-rpms
```

Comment:
Don't know whether its make sense, but this could be combined to a single RUN if you move `{{??}}` one line above, i.e. take care about the newlines

e.g. 

```
{{? fp.param.base === "rhel"}}
RUN yum-config-manager --enable rhel-server-rhscl-7-rpms && \{{??}}
RUN yum install -y centos-release-scl && \{{?}}
    yum install -y rh-maven33-maven
```